docs/integrate-new-apps/anpm-packaging.md

```diff
@@ -0,0 +1,109 @@
+# NPM package
+
+## Preliminary Checklist:
+
+### Researcher
+- Ensure the app code is structured appropriately.
+- Gather the necessary codebase for integration.
+
+### Developer
+This is on the APP SIDE
+
+- If the web app bundles external html, scripts, or strings:
+  - Install @rollup/plugin-html and rollup-plugin-string.
+- Verify package.json is configured correctly, especially the "name" field.
+- Check both Webpack and Rollup configurations.
+- Create staging and production hosting targets on gse-roar-assessment.
```

Comment:
- From the gse-roar-assessment project, navigate to "Hosting" on the left hand side.
- Click "add another site" in the top right corner
- Name the hosting targets with the following convention:
  - [roar][roam][roav]-appName
  - [roar][roam][roab]-appName-staging